prep-javascript-for-loops/main.js

```diff
@@ -0,0 +1,5 @@
+// create your loops here.
+var i = 100;
+for (i = 100; i >= 0; i--) {
+  console.log('Time till explosion ', i + '!');
+}
```

Comment:
Hi Changhyun, please write all of your loops in this file.

Reply:
hi Tim, do I need to resubmit this?? If so, do I need to fix in visual studio code and redo the git push  origin code?? Please let me know
Thanks!

Reply:
nvm i found how to resubmit it
thanks!